src/Components/__stories__/Icons.story.tsx

```diff
@@ -12,6 +13,54 @@ storiesOf("Components/Icons", module).add("All Icons", () => {
 
   return (
     <div style={{ margin: "20px" }}>
```

Comment:
Might be better to use palette here and down. 

```jsx
import { Row, Col } from 'Styleguide/Elements/Grid'`

<Box m={2}>
 <Row>
    <Col p={1}>...</Col>
 </Row>
</Box>
```

Reply:
@damassi Let me know if I'm setting this up in a weird way. Had some trouble getting the column elements to wrap but seems to look all right in Storybook now.

Reply:
One suggestion would be to just use flexbox (+ `flexWrap` property) and ditch the columns completely:

```js
const Icon = ({ children }) => <Box width='auto' ...other props...>{children}</Box> 
const icons = [Foo, Bar, Baz]

<Flex flexWrap="wrap">
  {icons.map(icon => <Icon>{icon}</Icon>}
</Flex>
```

Reply:
I'm encountering some difficultly aligning the icons to display in a neat grid with that approach. I'll start with this for now.